ui/css/treeherder-navbar-panels.css

```diff
@@ -24,6 +24,19 @@
  *  Active Filters panel
  */
 .active-filters-bar {
-    padding: 0 15px 0 15px;
-    border: 1px solid transparent;
```

Comment:
I pulled this border, I could be wrong but it didn't appear to really be doing anything.